src/core/server/saved_objects/migrationsv2/integration_tests/7_13_0_unknown_types.test.ts

```diff
@@ -47,6 +47,7 @@ describe('migration v2', () => {
           // `foo` documents
           // `space` documents (to mimic a migration with disabled plugins)
           dataArchive: Path.join(__dirname, 'archives', '7.13.0_with_unknown_so.zip'),
+          esArgs: ['xpack.security.enabled=false'],
```

Comment:
[o.e.b.BootstrapChecks] [node-01] Transport SSL must be enabled if security is enabled on a [basic] license. Please set [xpack.security.transport.ssl.enabled] to [true] or disable security by setting [xpack.security.enabled] to [false]